### ISSUE TITLE
2.13 ember init wipes bower.json (filesToRemove doesn't prompt before deleting)

### ISSUE THREAD

**Issue by ibroadfo** [ember-cli/ember-cli#6876]
When upgrading to 2.13, running `ember init` removes the `bower.json` file without prompting despite it containing dependencies.

Steps to reproduce:

`cat bower.json`:
```
{
  "name": "orange-whisky",
  "dependencies": {
    "remodal": "1.1"
  }
}
```
---
`ember init`
```
installing app
? Overwrite .travis.yml? Yes, overwrite
? Overwrite app/templates/application.hbs? Yes, overwrite
? Overwrite config/environment.js? Yes, overwrite
? Overwrite ember-cli-build.js? Yes, overwrite
? Overwrite package.json? Yes, overwrite
? Overwrite testem.js? Yes, overwrite
? Overwrite tests/helpers/start-app.js? Yes, overwrite
  identical .bowerrc
  identical .editorconfig
  identical .ember-cli
  overwrite .travis.yml
  identical .watchmanconfig
  identical README.md
  identical app/app.js
  identical app/index.html
  identical app/resolver.js
  identical app/router.js
  overwrite app/templates/application.hbs
  overwrite config/environment.js
  create config/targets.js
  overwrite ember-cli-build.js
  identical .gitignore
  overwrite package.json
  identical public/crossdomain.xml
  identical public/robots.txt
  overwrite testem.js
  identical tests/helpers/destroy-app.js
  identical tests/helpers/module-for-acceptance.js
  identical tests/helpers/resolver.js
  overwrite tests/helpers/start-app.js
  identical tests/index.html
  identical tests/test-helper.js
  remove bower.json
NPM: Installed dependencies
installing ember-cli-eslint
  create .eslintrc.js
  create tests/.eslintrc.js

The following JSHint config files were found:
  .jshintrc
  tests/.jshintrc
? What would you like to do? Remove them all
```

---
`git status -s bower.json`
```
 D bower.json
```

---

Output from `ember version --verbose && npm --version`:
```
ember-cli: 2.13.0-beta.1
http_parser: 2.7.0
node: 7.4.0
v8: 5.4.500.45
uv: 1.10.1
zlib: 1.2.8
ares: 1.10.1-DEV
modules: 51
openssl: 1.0.2j
icu: 58.2
unicode: 9.0
cldr: 30.0.3
tz: 2016j
os: darwin x64
4.0.5
```

**Comment by rwjblue** [comment]
Thanks for reporting! Definitely seems fishy.

**Comment by rwjblue** [comment]
@kellyselden - Can you take a look at this?

**Comment by kellyselden** [comment]
This appears to be as designed. The removals go through our normal `destroy` blueprint logic, which don't ask for confirmation, just delete.

That said, there's nothing stopping this from getting smarter and prompting for all file removals in this new list.

**Comment by ibroadfo** [comment]
I'd definitely vote for a change to the design here, it seems like a pretty clear data loss scenario.

**Comment by rwjblue** [comment]
Agreed. This is very surprising.

**Comment by vlascik** [comment]
same here, this really shouldn't happen.

**Comment by mariogintili** [comment]
Ran into this too 😄  - Just to clarify. If the `bower.json` file is gone from the blueprint:

- Should users be able to select wether or not to use bower? As right now it looks like it's a fixed decision towards no, but seems like a bug from what I read.

- If new versions of ember-cli won't ship bower. What should we do went we want to add a dependency thats only available in bower? My use case was adding `bootstrap` - in the same fashion as the guides suggest: https://ember-cli.com/asset-compilation#stylesheets

**Comment by Turbo87** [comment]
@mariogintili tl;dr is that bower is still supported by Ember CLI, just not used for the default set of addons that we provide in our blueprints. bower will not be installed by default, but if you use it right now for some stuff you can just continue doing that.

**Comment by mariogintili** [comment]
@Turbo87  cool thanks made the question(s) so people that come across this issue can keep working accordingly 😄

**Comment by ibroadfo** [comment]
Is this a blocker for 2.13?

**Comment by Turbo87** [comment]
@kellyselden what do you think?

**Comment by kellyselden** [comment]
I think so. We need a prompt for our files marked for deletion. We could revert the bower deletions until then.

**Comment by Turbo87** [comment]
okay, so the way forward here to unblock the Ember CLI release is: on the `beta` branch: remove `bower.json` from the [list of files that will always be removed](https://github.com/ember-cli/ember-cli/blob/b02d6abd002eee50507e85f1e4273599bb056d44/blueprints/app/index.js#L14) and the same for the `addon` blueprint

at a later point we can revisit this topic on the `master` branch and add a better prompt for this

**Comment by kellyselden** [comment]
and .bowerrc

**Comment by kellyselden** [comment]
Fixed via #6975
